### PR TITLE
docs: Add jobs tolerations attribute in helm chart docs

### PR DESCRIPTION
**Pull request by dtotopus**
Add attribute definition for child chart Job tolerations

**Reason for Change**:
Current documentation doesn't specify that in order for tolerations to work child chart requires tolerations to be defined. Attribute isn't listed in the README.md

**Is this a chart or deployment yaml update?**
Documentation update only

**Does this change contain code from or inspired by another project?**
No

### CODE REVIEW

File: charts/csi-secrets-store-provider-azure/README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
...
 | `secrets-store-csi-driver.linux.kubeletRootDir`                  | Configure the kubelet root dir                                                                                                                                                                        | `/var/lib/kubelet`                                                                               |
 | `secrets-store-csi-driver.linux.metricsAddr`                     | The address the metric endpoint binds to                                                                                                                                                              | `:8080`                                                                                          |
+| `secrets-store-csi-driver.linux.tolerations`                     | Tolerations for the Jobs on linux nodes                                                                                                                                                              | `[]`                                                                                          |
```
Comment (nilekhc) [inline charts/csi-secrets-store-provider-azure/README.md:109]:
```suggestion
| `secrets-store-csi-driver.linux.tolerations`                     | Tolerations for linux nodes                                                                                                                                                              | `[]`                                                                                          |
```

File: charts/csi-secrets-store-provider-azure/README.md [hunk 2]
```diff
@@ -121,6 +122,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
...
 | `secrets-store-csi-driver.windows.kubeletRootDir`                | Configure the kubelet root dir                                                                                                                                                                        | `C:\var\lib\kubelet`                                                                             |
 | `secrets-store-csi-driver.windows.metricsAddr`                   | The address the metric endpoint binds to                                                                                                                                                              | `:8080`                                                                                          |
+| `secrets-store-csi-driver.windows.tolerations`                     | Tolerations for the Jobs on windows nodes                                                                                                                                                              | `[]`                                                                                          |
```
Comment (nilekhc) [inline charts/csi-secrets-store-provider-azure/README.md:125]:
```suggestion
| `secrets-store-csi-driver.windows.tolerations`                     | Tolerations for windows nodes                                                                                                                                                              | `[]`                                                                                          |
```

**Comment by nilekhc** [comment]
@dtotopus Thanks for opening the PR. I have left couple nit.

Could you move these changes to [manifest_staging](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/manifest_staging/charts/csi-secrets-store-provider-azure/README.md) dir. We promote this to `charts` after new helm chart release.

Also, could you add [semantic](https://github.com/Ezard/semantic-prs) commit?

**Comment by dtotopus** [comment]
Thanks a lot for the comments! Updated the PR

File: manifest_staging/charts/csi-secrets-store-provider-azure/README.md [hunk 3]
```diff
@@ -121,6 +122,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
...
 | `secrets-store-csi-driver.windows.kubeletRootDir`                | Configure the kubelet root dir                                                                                                                                                                        | `C:\var\lib\kubelet`                                                                             |
 | `secrets-store-csi-driver.windows.metricsAddr`                   | The address the metric endpoint binds to                                                                                                                                                              | `:8080`                                                                                          |
+| `secrets-store-csi-driver.windows.tolerations`                   | Tolerations for linux nodes                                                                                                                                                                           | `[]`                                                                                             |
```
Comment (aramase) [inline manifest_staging/charts/csi-secrets-store-provider-azure/README.md:125]:
```suggestion
| `secrets-store-csi-driver.windows.tolerations`                   | Tolerations for driver pods on windows nodes                                                                                                                                                                           | `[]`                                                                                             |
```

File: manifest_staging/charts/csi-secrets-store-provider-azure/README.md [hunk 4]
```diff
@@ -106,6 +106,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
...
 | `secrets-store-csi-driver.linux.kubeletRootDir`                  | Configure the kubelet root dir                                                                                                                                                                        | `/var/lib/kubelet`                                                                               |
 | `secrets-store-csi-driver.linux.metricsAddr`                     | The address the metric endpoint binds to                                                                                                                                                              | `:8080`                                                                                          |
+| `secrets-store-csi-driver.linux.tolerations`                     | Tolerations for linux nodes                                                                                                                                                                           | `[]`                                                                                             |
```
Comment (aramase) [inline manifest_staging/charts/csi-secrets-store-provider-azure/README.md:109]:
```suggestion
| `secrets-store-csi-driver.linux.tolerations`                     | Tolerations for driver pods on linux nodes                                                                                                                                                                           | `[]`                                                                                             |
```

**Review by aramase — APPROVED** [review]

**Pull request merged**